agent/transformers/graphite_test.go

```diff
@@ -87,6 +87,47 @@ func TestParseGraphite(t *testing.T) {
 	}
 }
 
+func TestParseGraphiteTags(t *testing.T) {
+	assert := assert.New(t)
+
+	testCases := []struct {
+		metric         string
+		expectedFormat GraphiteList
+	}{
+		{
+			metric: "metric.value 1 123456789",
+			expectedFormat: GraphiteList{
+				{
+					Path:      "metric.value",
+					Value:     1,
+					Timestamp: 123456789,
+					Tags: []*types.MetricTag{
+						&types.MetricTag{
+							Name:  "instance",
+							Value: "hostname",
+						},
+					},
+				},
+			},
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(tc.metric, func(t *testing.T) {
+			event := types.FixtureEvent("test", "test")
+			event.Check.Output = tc.metric
+			event.Check.OutputMetricTags = []*types.MetricTag{
+				{
+					Name:  "instance",
+					Value: "hostname",
+				},
+			}
+			graphite := ParseGraphite(event)
+			assert.Equal(tc.expectedFormat, graphite)
+		})
+	}
+}
```

Comment:
It might be useful to refactor this test so that `outputMetricTags []*types.MetricTag` is a field in the `testCases` struct. This would allow you to dynamically configure `event.Check.OutputMetricTags = tc.outputMetricTags` and add more test cases (ex. multiple tags, empty tags, nil tags, tags with empty name, tags with empty value) rather than testing just the happy path.